src/components/tabs/tabs.scss

```diff
@@ -178,6 +184,7 @@ md-tabs-content-wrapper {
   right: 0;
   bottom: 0;
   overflow: hidden;
+  z-index: 1;
```

Comment:
Our z-indexes are always defined in the `variables.scss` file. Please take a look at those. We are having a strict use-case of z-indexes. Applying random z-indexes may break some other components.